lldb/source/API/SBValue.cpp

```diff
@@ -1210,15 +1210,16 @@ bool SBValue::GetDescription(SBStream &description) {
 
   ValueLocker locker;
   lldb::ValueObjectSP value_sp(GetSP(locker));
-  if (value_sp) {
-    DumpValueObjectOptions options;
-    options.SetUseDynamicType(m_opaque_sp->GetUseDynamic());
-    options.SetUseSyntheticValue(m_opaque_sp->GetUseSynthetic());
-    value_sp->Dump(strm, options);
-  } else {
+  if (!value_sp) {
     strm.PutCString("No value");
```

Comment:
OMG, that is horrible that we return false and yet still put "No value" into the stream???

Reply:
I wasn't sure if the `No value` output was an expected behavior on the SBValue, so I didn't want to change that behavior.

For example:

```
$ lldb
(lldb) script
Python Interactive Interpreter. To exit, type 'quit()', 'exit()' or Ctrl-D.
>>> lldb.SBValue() # __str__ > __repr__ > SBValue::GetDescription
No value
```

If I removed the `No value` string, then GetDescription would simply return "" or maybe None. I can update that if its appropriate, but I wasn't sure if that behavioral change was desired.

Reply:
I reverted this change and instead just check `IsValid()` before we call into `GetDescription()`.